### PR TITLE
Convert UI

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: lib/app/shared/styles/amount_field_decoration.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/material.dart';
+
+import 'base_field_decoration.dart';
+
+InputDecoration amountFieldDecoration = baseFieldDecoration.copyWith(
+  hintText: 'Amount',
```
Comment (VovaMedvediev) [inline lib/app/shared/styles/amount_field_decoration.dart:6]: Should it be internalized?

File: lib/app/shared/plugins/convert/view/convert.dart [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+              currencies: state.fromList,
+              onChanged: (value) => notifier.updateFrom(value!),
+            ),
+            const SpaceH10(),
+            ConvertSwitchButton(
+              onChanged: () => notifier.switchFromTo(),
+            ),
+            const SpaceH20(),
+            const ConvertTitleText(
+              text: 'To',
```
Comment (VovaMedvediev) [inline lib/app/shared/plugins/convert/view/convert.dart:50]: intl?

File: lib/app/shared/plugins/convert/view/convert.dart [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      appBar: AppBar(
+        title: const Text('Convert'),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(10.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            const ConvertTitleText(
+              text: 'From',
```
Comment (VovaMedvediev) [inline lib/app/shared/plugins/convert/view/convert.dart:37]: intl?

File: lib/app/shared/plugins/convert/view/components/request_quote_button.dart [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  Widget build(BuildContext context) {
+    return TextButton(
+      onPressed: onPressed,
+      style: ButtonStyle(
+        backgroundColor: MaterialStateProperty.all<Color>(
+          const Color(0xFFF5F5F5),
+        ),
+      ),
+      child: const Text(
+        'Request Quote',
```
Comment (VovaMedvediev) [inline lib/app/shared/plugins/convert/view/components/request_quote_button.dart:21]: intl?

File: lib/app/shared/styles/amount_field_decoration.dart (hunk 1 above)
Comment (eli1stark) [reply lib/app/shared/styles/amount_field_decoration.dart:6]: Considered to skip intl until final design will be in place

**Pull request merged**